src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,61 @@
+package duke;
+
+/**
+ * The Parser class deals with making sense of the user command.
+ */
+public class Parser {
+
+    /**
+     * Reads the user input and determines which command to execute.
+     * @param command User input.
+     * @return Command The type of command to execute.
+     * @throws DukeException
+     */
+    public static Command parse(String command) throws DukeException {
```

Comment:
This method is more than 30 lines and very lengthy, perhaps you can consider shortening it to improve readability?

Reply:
Alright thanks